test/runner.js

```diff
@@ -1,8 +1,45 @@
 "use strict";
 
-require('should');
-var wordpressLib = require(__dirname + '/../lib/wordpress.js');
+var xmlrpc = require('xmlrpc'),
+    url = require('url'),
+    should = require('should'),
+    wordpressLib = require(__dirname + '/../lib/wordpress.js');
 
 describe('Wordpressorg', function () {
+  describe('Client', function () {
+    var settings,
+        blog,
+        url = 'http://somedomain.com';
 
-});
\ No newline at end of file
+    it('should create Client when not given http auth entries are undefined', function (done) {
+      settings = {
+        url: url,
+        username: 'fake_user',
+        password: 'password',
+        blogId: 1,
+        httpauthuser: undefined,
+        httpauthpassword: undefined
+      };
+      blog = wordpressLib.createClient(settings);
+      blog.should.not.equal(null);
+      blog.rpc.should.not.equal(null);
+      blog.rpc.options.should.not.equal(null);
```

Comment:
It's more about adding assertions that ensure the client is created and behaves as expected as a result rather than doing null checking (which has some value, but making sure your parameters are properly processed has more).